.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,17 @@
+blank_issues_enabled: false
```

Comment:
Does this mean that we can't use issues for tracking of tasks, or is this only for external contributors?

Reply:
This is only meant for external contributors.